src/core/Object3D.js

```diff
@@ -615,6 +631,44 @@ Object3D.prototype = Object.assign( Object.create( EventDispatcher.prototype ),
 
 	},
 
+	updateWorldMatrix: function ( updateParents, updateChildren ) {
```

Comment:
Um, it can be very confusing for users when they encounter this method in the source code. I think it would be better for now to turn `updateWorldMatrix()` into a function and put it in the closure of lookAt().

Reply:
Every method in the library that calls `updateMatrixWorld()` expecting it to work properly needs to call the new method instead.

That would be in the next PR.

Reply:
Um, I'm not sure we are heading the right way with introducing a new method. At least, we should not rush with this change and think about the consequences.

Reply:
I have purposely not touched `updateMatrixWorld()` with this PR.